.github/workflows/check.yaml

```
@@ -189,12 +189,7 @@ jobs:
           # Create the path that is expected to have a coverage.xml for tics
           mkdir -p tests/report/
 
-          coverage_files=""
-          for file in ./artifacts/coverage-*; do
-            if [ -f "$file" ]; then
-              coverage_files="$coverage_files $file"
-            fi
-          done
+          coverage_files=$(echo ./artifacts/coverage-*)
```

Does it work with a simple glob, removing the echo middleman?

```suggestion
          coverage_files=./artifacts/coverage-*
```

---

Now that the files are hidden, using echo is necessary. Before it was possible to remove the echo

---

why is it necessary for hidden files? Doesn't this work: `echo ./artifacts/.coverage-*`?

---

Yes, it works. I don't know exactly what I did before, but it was not working, Leaving the parenthesis and removing the echo works.

---

It doesn't work for me keeping the parentheses. It should work without the parens - see https://github.com/canonical/charmed-openstack-upgrader/pull/613/commits/401a26e759a1615d0412bced7807e684e01376a5#r1988095978